setup.py

```diff
@@ -25,7 +25,7 @@
     package_dir={"atlassian": "atlassian"},
     include_package_data=True,
     zip_safe=False,
-    install_requires=["deprecated", "requests", "six", "oauthlib", "requests_oauthlib", "jmespath"],
+    install_requires=["deprecated", "requests", "six", "oauthlib", "requests_oauthlib", "jmespath", "bs4"],
```

Comment:
The PyPi package is called `beautifulsoup4`, isn't it?

```suggestion
    install_requires=["deprecated", "requests", "six", "oauthlib", "requests_oauthlib", "jmespath", "beautifulsoup4"],
```

Reply:
yeah, I thought it failed on tox with `beautifulsoup4` but it now it's working

Reply:
https://github.com/atlassian-api/atlassian-python-api/commit/450700f03bc7bd353d5e72f97a6717db98fd0717

I think `bs4` is the correct package name, while `beautifulsoup4` is the project name. I had an issue with the recent hotfix